tests/fixtures/bundlePage.ts

```diff
@@ -1,50 +1,89 @@
-import {expect, test as base } from '@playwright/test'
-import type {Page } from '@playwright/test'
+import { expect, test as base } from '@playwright/test'
+import type { Page } from '@playwright/test'
 
-const getBundleName = prefix => browser => prefix + '_' + browser
+const getBundleName = (prefix: string, suffix?: string) => (browser: string) =>
+  prefix + '_' + browser + `${suffix ? '_' + suffix : ''}`
+
+const dialogTitle = 'Copy addon settings to your new staging bundle?'
 
 class BundlePage {
-  constructor(
-    public readonly page: Page,
-    public readonly browserName: String,
-  ) {}
+  constructor(public readonly page: Page, public readonly browserName: String) {}
 
   async goto() {
     await this.page.goto('/settings/bundles')
+    // wait for the page to load
+    await expect(this.page.getByRole('button', { name: 'Bundles' })).toBeVisible()
   }
 
   async createBundle(name: string) {
-    await this.goto()
-    await this.page.goto('/settings/bundles')
     await this.page.getByRole('button', { name: 'add Add Bundle' }).click()
     await this.page.getByRole('main').getByRole('textbox').fill(name)
     await this.page.getByRole('button', { name: 'Select an option...' }).click()
     const launcher = '1.0.2'
     await this.page.getByText(launcher).click()
+    // check that bundle is compatible
+    const compatibleMessage = 'Checks complete: Bundle is compatible'
+    await expect(this.page.getByText(compatibleMessage)).toBeVisible()
```

Comment:
This line fails on the second bundle creation and I have no idea why. It seems that bundle title and launcher are not set correctly so compatibility checks fail. Any ideas @flynput ?

Reply:
I'm still checking it. The 'anomaly' seems to be caused by a form reset, but I can't pinpoint yet what causes that.

Reply:
@Innders Fixed the issue, please check the commit. The problem was caused by addons fetching which reset the form. I changed the logic a bit to keep the name/installer version when that happens.